ingestr/src/asana_source/__init__.py

```diff
@@ -0,0 +1,264 @@
+"""
```

Comment:
I see that we dont add `_source` to the package for other platforms, even when they are only source. Maybe we keep that for consistency?

Reply:
naming the package to `asana` causes ambiguity in imports when importing the `asana` client.

This was originally called `asana` package, but when importing `asana` client, it would cause a circular import.

dlt also ran into the same issue, and as a consequence, their version is called `asana_dlt`